src/networking.c

```diff
@@ -41,11 +41,40 @@
 #include <math.h>
 #include <ctype.h>
 #include <stdatomic.h>
+#include "intset.h"
+#include <stdbool.h>
```

Comment:
I don't see a strong reason to use bool here, I can make do with 0. I'm new to the project, so that's why I used bool I guess. Switching it to int function for now.

Reply:
But, I think it's still good to have a conversation around it. It can potentially make code clearer I guess and helpful in few usecases as well.

Reply:
Yes, I agree with you.